packages/component/src/Attachment/Assets/DownloadIcon.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 
-export default ({ className, size = 1 }) =>
+import { localize } from '../../Localization/Localize';
+
+export default ({ className, language, size = 1 }) =>
   <svg
+    aria-label={ localize('Download file', language) }
```

Comment:
Since this seems to be a presentational component, it may make more sense to do the localization up a level and just pass a `label` prop instead. That will give this component a bit more flexibility.